convert2rhel/pkghandler.py

```diff
@@ -64,15 +64,15 @@ def call_yum_cmd_w_downgrades(cmd, pkgs):
             break
         # handle error condition
         loggerinst.info("Resolving dependency errors ... ")
-        output = resolve_dep_errors(output, [])
+        output = resolve_dep_errors(output)
         # if we have problematic packages, remove them
-        problematic_pkgs = get_problematic_pkgs(output, [])
+        problematic_pkgs = get_problematic_pkgs(output)
         if problematic_pkgs["errors"]:
             loggerinst.warning(
                 "Removing problematic packages to continue with conversion:\n%s" %
                 "\n".join(problematic_pkgs["errors"])
             )
-            utils.remove_pkgs(problematic_pkgs['errors'], critical=False)
+            utils.remove_pkgs(problematic_pkgs['errors'], backup=False, critical=False)
```

Comment:
Couldn't find it mentioned, why do we remove backups? Seems like a mistake

Reply:
I briefly mentioned it in the PR description: _Also, the previous implementation was, before removing them, trying to backup the packages that were causing dependency errors. The backup makes no sense in this stage as there's no rollback to happen anymore._
The backup is not even possible since the original CentOS/OL repos are already removed. So we don't have where to download the package backup from.
And, actually, there's a customer case where the conversion got stuck at exactly this point - on the package backup.